examples/rsp-next-ts/next.config.js

```diff
@@ -46,10 +46,12 @@ const withTM = require("next-transpile-modules")([
   "@react-spectrum/well",
   "@spectrum-icons/illustrations",
   "@spectrum-icons/ui",
-  "@spectrum-icons/workflow"
+  "@spectrum-icons/workflow",
```

Comment:
why were the trailing commas on the last items added?

Reply:
from prettier, just ran on all files in the project for consistency